.github/ISSUE_TEMPLATE/orientation-epic.md

```diff
@@ -14,30 +14,43 @@ As a new VA.gov team member, I need to complete onboarding activities so I can s
 - [ ] Each team member has completed each item in the orientation checklist below.
 
 ## Onboarding Checklist
+### Share your info with others
```

Comment:
This section is intended to be a field input. So when the new VFS team member submits this ticket, they should be adding their name, role, email, etc. Can we edit, "Share your info with others" to "Please input the following info:"? And then can we also edit the fields to look like a field input? I'm open to ideas.